roles/openshift_master/templates/atomic-openshift-master-controllers.service.j2

```diff
@@ -7,7 +7,7 @@ Before={{ openshift.common.service_type }}-node.service
 Requires=network.target
 
 [Service]
-Type=simple
+Type=notify
```

Comment:
We will probably need to conditionalize this based on the version, right? If version < supported version then Type=simple else type=notify